live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/PdfOperationsIntegrationTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.live_ingester.domain.usecase.e2e;
+
+import org.junit.jupiter.api.Test;
+
+import org.alfresco.hxi_connector.live_ingester.util.E2ETestBase;
+
+public class PdfOperationsIntegrationTest extends E2ETestBase
+{
+
+    @Test
+    void testPdFOperations()
+    {
+
+        // given
+        containerSupport.prepareSFSToReturnSuccess();
+        containerSupport.prepareHxIToReturnSuccessWithStorageLocation();
+        containerSupport.prepareHxIToReturnSuccessAfterReceivingFileLocation();
+
+        // when
+        String atsBody = """
+                {
+                    "targetReference": "e71dd823-82c7-477c-8490-04cb0e826e66"
+                }""";
+        containerSupport.raiseATSEvent(atsBody);
+
+        // then
+        containerSupport.expectSFSMessageReceived();
```

Comment:
Maybe `expectSFSRequestReceived`? Can we also check the request contains the target reference from above?

Reply:
Target reference is a part of GET request URL path.  Should we extract it ?

Reply:
updated in [b008f17](https://github.com/Alfresco/hxinsight-connector/pull/142/commits/b008f179bd306a7b86f0d34cbdb06804198b8a2c)